.github/workflows/create_scripts.yml

```diff
@@ -3,10 +3,9 @@ name: build_macos
 on:
   push:
     paths:
-      - '.github/workflows/build_deps.yml'
+      - '.github/workflows/master.yml'
```

Comment:
There is no workflow with this name.  Is that intentional?  What does the `create_scripts.yml` do now, anyway?

Reply:
Right, it should say `main` - will push a fix in a bit.

The script now reads the dependency config in the workflow and updates the global build script, so we don't have to update the versions/hashes in two files.

The global build script is convenience for local builds, so I thought this would make maintenance convenient for us as well as keeping that script up2date whenever we do a release.

Reply:
Ah, so now it reads https://github.com/obsproject/obs-deps/pull/60/files#diff-7829468e86c1cc5d5133195b5cb48e1ff6c75e3e9203777f6b2e379d9e4882b3R23-R56 and updates https://github.com/obsproject/obs-deps/pull/60/files#diff-c1f2aefe3a1708caf92d65020c7b456155cb3c8b8f7dbc69a8f0cb1076130893R25-R42 ?  Is that about it?  Does that necessitate having the exact same deps for each OS we build for?

Reply:
Not in that specific case, because the script uses the values set as `env` in the OS-specific build job. So if you specify different versions for Windows, it wouldn't change a thing.

It might trip up the standalone scripts' fallbacks (as they are read from the workflow and take the first value they encounter), but I'll pass that bridge once we get there. I'll probably have to use more complex parsing in Powershell anyway so that becomes a non-issue.

Reply:
Sounds good for now.

Reply:
Should be fixed with a recent force-push.